pkg/utils/hex/hex.go

```diff
@@ -0,0 +1,55 @@
+package hex
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+// EnsurePrefix adds the prefix (0x) to a given hex string.
+func EnsurePrefix(str string) string {
+	if !strings.HasPrefix(str, "0x") {
+		str = "0x" + str
+	}
+	return str
+}
+
+// ToBig parses the given hex string or panics if it is invalid.
+func ToBig(s string) *big.Int {
+	n, ok := new(big.Int).SetString(s, 16)
+	if !ok {
+		panic(fmt.Errorf(`failed to convert "%s" as hex to big.Int`, s))
+	}
+	return n
+}
+
+// RemovePrefix removes the prefix (0x) of a given hex string.
+func RemovePrefix(str string) string {
```

Comment:
We could align with the std lib naming here. (e.g. `strings.TrimPrefix`)
```suggestion
// TrimPrefix removes the prefix (0x) of a given hex string.
func TrimPrefix(str string) string {
```